class TestFailure(Exception):
    pass

class DependencyFailure(Exception):
    pass
